modules/apps/portal-search-solr/portal-search-solr/src/main/resources/META-INF/resources/schema.xml

```diff
@@ -493,7 +493,7 @@
 		<field indexed="true" name="classTypeId" stored="true" type="string" />
 		<field indexed="true" name="comments" stored="true" type="text" />
 		<field indexed="true" name="companyId" stored="true" type="long" />
-		<field indexed="true" name="configurationModelAttributeName" stored="true" type="string" />
+		<field indexed="true" multiValued="true" name="configurationModelAttributeName" stored="true" type="string" />
```

Comment:
It configured the schema for solr, the other search engine we support.
Solr isn't as intelligent about multi-value fields.

Sent from my mobile device.